docs/lambda.asciidoc

```diff
@@ -1,150 +1,86 @@
-:framework: lambda
-
 [[lambda]]
 
 ifdef::env-github[]
 NOTE: For the best reading experience,
 please view this documentation at https://www.elastic.co/guide/en/apm/agent/nodejs/current/lambda.html[elastic.co]
 endif::[]
 
-=== Get started with Lambda
-
-experimental::[]
+=== Get started with AWS Lambda Node.js Functions
+:layer-section-type: with-agent
 
-Getting Elastic APM set up for your lambda functions is easy,
-and there are various ways you can tweak it to fit your needs.
-Follow the guide below to get started, and for more advanced topics,
-check out the <<api,API Reference>>.
+The Node.js APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
 
 [float]
-[[lambda-installation]]
-==== Installation
-
-Setting up your Lambda function is a two-step process.
+[[aws-lambda-nodejs-prerequisites]]
+==== Prerequisites
 
-1. <<lambda-extension>>
-2. <<lambda-instrumenting>>
-
-First, https://github.com/elastic/apm-aws-lambda/blob/main/docs/aws-lambda-extension.asciidoc[install the Elastic
-APM Lambda extension] into your Lambda environment. Elastic uses a Lambda
-extension to forward data to APM Server in a way that does not interfere with the
-execution of your Lambda function.
+You need an APM Server to send APM data to. Follow the {apm-guide-ref}/apm-quick-start.html[APM Quick start] if you have not set one up yet. For the best-possible performance, we recommend setting up APM on {ecloud} in the same AWS region as your AWS Lambda functions.
 
 [float]
-[[lambda-extension]]
-==== Install the Elastic APM Lambda extension
-
-Elastic uses a Lambda extension to forward data to APM Server in a way that does not interfere with the
-execution of your Lambda function.
+[[aws-lambda-nodejs-quick-start]]
+==== Quick Start
 
-See the https://github.com/elastic/apm-aws-lambda/blob/main/docs/aws-lambda-extension.asciidoc[installation documentation] to get started.
-
-Compatibility note: If using v3.30.0 or later of the APM Node.js Agent, one
-must use v0.0.4 or later of the APM Lambda extension.
-
-Once you have the extension installed in your environment, you can
-configure your Lambda handler to instrument your function.
+To get started with APM for your Node.js AWS Lambda functions, follow the steps below.
```

Comment:
```suggestion
To get started with APM for your Node.js AWS Lambda functions follow the steps below.
```

Reply:
Done in https://github.com/elastic/apm-agent-nodejs/pull/2630/commits/94da3631aaf5d09a68b93024b24c29c9f2c72f04